content/02.main-text.md

```diff
@@ -1,3 +1,21 @@
 ## Main text {.page_break_before}
 
-*Work in progress*
+### Introduction
+
+Open research – which includes sharing code, data, and manuscripts – benefits the researchers who engage in open practices [@doi:10.7554/eLife.16800], their scientific peers, and the public.
+`TODO: more references needed`
+Here we advocate for the benefits of writing review articles openly, specifically planning, organizing, writing, and editing this type of manuscript collaboratively in an unrestricted public forum.
+Reviews presenting the state of the art in a scientific field are often prepared by a single research group or a small team of colleagues.
+In contrast, broadly opening the writing process to anyone engaged in the topic can help maximize the review's value by ensuring that it represents diverse opinions and does not neglect relevant research.
+Review authors can engage with the authors of original research to clarify their methods and results and present them accurately.
```

Comment:
Out outline has `Versioning/provenance of metadata (issues, pull requests, etc) (We can refer to or implement? strategies for backup up a GitHub repo https://help.github.com/articles/backing-up-a-repository/)` so we'll definitely want to return to this issue.  I'll link with a TODO about archiving for now.